ansibullbot/triagers/plugins/shipit.py

```diff
@@ -261,6 +261,14 @@ def get_shipit_facts(issuewrapper, inmeta, module_indexer, core_team=[], botname
     ]
 
     files = [f for f in iw.files if not f.startswith(u'changelogs/fragments/')]
+
```

Comment:
https://github.com/ansible/ansibullbot/blob/a06dab3fb4b710fb38cc4b09894900a8f6bb8467/ansibullbot/triagers/plugins/shipit.py#L113 Shouldn't the above be also changed to allow for `test/sanity/ignore.txt`? Won't it fail with `automerge !module file(s) test failed` currently?